index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/9f61ce6d5c.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="css/fontawesome/css/font-awesome.min.css" />
+    <link
+      rel="stylesheet"
+      href="https://bootswatch.com/4/yeti/bootstrap.min.css"
+    />
+    <title>MyLibrary App</title>
+  </head>
+  <body>
+    <div class="container mt-4">
+      <h1 class="display-4 text-center">
+        <i class="fas fa-book-open text-primary"></i> My<span
+          class="text-primary"
+          >Library</span
+        >Store
+      </h1>
+      <form id="book-form">
+        <div class="form-group">
+          <label for="title">Title</label>
+          <input type="text" id="title" class="form-control" />
+        </div>
+        <div class="form-group">
+          <label for="author">Author</label>
+          <input type="text" id="author" class="form-control" />
+        </div>
+        <div class="form-group">
+          <label for="pages">Number of pages</label>
+          <input type="text" id="page" class="form-control" />
+        </div>
+        <input
+          type="submit"
+          value="Add Book"
+          class="btn btn-primary btn-block"
+        />
+      </form>
+      <table class="table table-striped mt-5">
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>Author</th>
+            <th>Number of pages</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody id="book-list"></tbody>
```

Comment:
From your app;


According to the project requirement, you're supposed to have a toggle feature, where users can change the book status.

![image](https://user-images.githubusercontent.com/31897434/87005323-bbf7a880-c1b6-11ea-8d55-43d445f87e8c.png)

https://www.theodinproject.com/courses/javascript/lessons/library